package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-simple-auth-token",
-  "version": "5.0.0",
+  "version": "5.0.1",
```

Comment:
Should we bump minor instead of patch?

Reply:
That makes sense. I will update it to 5.1.0

Reply:
This has been updated.